<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MSDFit</class>
 <widget class="QWidget" name="MSDFit">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>794</width>
    <height>785</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_5" stretch="0,1,0,0">
   <item>
    <widget class="QGroupBox" name="gbInput">
     <property name="title">
      <string>Input</string>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="0">
       <widget class="MantidQt::MantidWidgets::DataSelector" name="dsSampleInput" native="true"/>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="loMSDFit">
     <item>
      <layout class="QVBoxLayout" name="loMSDPlot" stretch="0">
       <item>
        <layout class="QHBoxLayout" name="loPlotProperties" stretch="0,0">
         <item>
          <layout class="QVBoxLayout" name="loProperties">
           <item>
            <layout class="QVBoxLayout" name="properties"/>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QVBoxLayout" name="loPlotsLayout" stretch="5,3,0">
           <property name="sizeConstraint">
            <enum>QLayout::SetNoConstraint</enum>
           </property>
           <item>
            <widget class="MantidQt::MantidWidgets::PreviewPlot" name="ppPlotTop" native="true">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
               <horstretch>0</horstretch>
               <verstretch>1</verstretch>
              </sizepolicy>
             </property>
             <property name="minimumSize">
              <size>
               <width>0</width>
               <height>125</height>
              </size>
             </property>
             <property name="showLegend" stdset="0">
              <bool>true</bool>
             </property>
             <property name="curveErrorBars" stdset="0">
              <stringlist>
               <string>Sample</string>
              </stringlist>
             </property>
             <property name="canvasColour" stdset="0">
              <color>
               <red>255</red>
               <green>255</green>
               <blue>255</blue>
              </color>
             </property>
            </widget>
           </item>
           <item>
            <widget class="MantidQt::MantidWidgets::PreviewPlot" name="ppPlotBottom" native="true">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
               <horstretch>1</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="minimumSize">
              <size>
               <width>0</width>
               <height>75</height>
              </size>
             </property>
             <property name="showLegend" stdset="0">
              <bool>true</bool>
             </property>
             <property name="canvasColour" stdset="0">
              <color>
               <red>255</red>
               <green>255</green>
               <blue>255</blue>
              </color>
             </property>
            </widget>
           </item>
           <item>
            <layout class="QVBoxLayout" name="verticalLayout">
             <property name="topMargin">
              <number>0</number>
             </property>
             <property name="bottomMargin">
              <number>0</number>
             </property>
             <item>
              <layout class="QHBoxLayout" name="horizontalLayout_2">
               <property name="topMargin">
                <number>0</number>
               </property>
               <item>
                <widget class="QLabel" name="lbPlotSpectrum">
                 <property name="text">
                  <string>Plot Spectrum:</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QSpinBox" name="spPlotSpectrum">
                 <property name="minimum">
                  <number>0</number>
                 </property>
                 <property name="maximum">
                  <number>0</number>
                 </property>
                </widget>
               </item>
               <item>
                <spacer name="horizontalSpacer_3">
                 <property name="orientation">
                  <enum>Qt::Horizontal</enum>
                 </property>
                 <property name="sizeType">
                  <enum>QSizePolicy::Expanding</enum>
                 </property>
                 <property name="sizeHint" stdset="0">
                  <size>
                   <width>40</width>
                   <height>20</height>
                  </size>
                 </property>
                </spacer>
               </item>
               <item>
                <widget class="QPushButton" name="pbSingleFit">
                 <property name="text">
                  <string>Fit Single Spectrum</string>
                 </property>
                </widget>
               </item>
               <item>
                <spacer name="horizontalSpacer_2">
                 <property name="orientation">
                  <enum>Qt::Horizontal</enum>
                 </property>
                 <property name="sizeHint" stdset="0">
                  <size>
                   <width>40</width>
                   <height>20</height>
                  </size>
                 </property>
                </spacer>
               </item>
               <item>
                <widget class="QPushButton" name="pbPlotPreview">
                 <property name="text">
                  <string>Plot Current Preview</string>
                 </property>
                </widget>
               </item>
               <item>
                <spacer name="horizontalSpacer">
                 <property name="orientation">
                  <enum>Qt::Horizontal</enum>
                 </property>
                 <property name="sizeHint" stdset="0">
                  <size>
                   <width>40</width>
                   <height>20</height>
                  </size>
                 </property>
                </spacer>
               </item>
               <item>
                <widget class="QCheckBox" name="ckPlotGuess">
                 <property name="text">
                  <string>Plot Guess</string>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
            </layout>
           </item>
          </layout>
         </item>
        </layout>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <widget class="MantidQt::CustomInterfaces::IDA::IndirectSpectrumSelectionView" name="svSpectrumView" native="true"/>
   </item>
   <item>
    <widget class="QGroupBox" name="gbOutput">
     <property name="title">
      <string>Output</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_8">
      <item>
       <widget class="QPushButton" name="pbPlot">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="text">
         <string>Plot Result</string>
        </property>
        <property name="checked">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer_1">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="pbSave">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="text">
         <string>Save Result</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>MantidQt::MantidWidgets::DataSelector</class>
   <extends>QWidget</extends>
   <header>MantidQtWidgets/Common/DataSelector.h</header>
  </customwidget>
  <customwidget>
   <class>MantidQt::MantidWidgets::PreviewPlot</class>
   <extends>QWidget</extends>
   <header>MantidQtWidgets/LegacyQwt/PreviewPlot.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>MantidQt::CustomInterfaces::IDA::IndirectSpectrumSelectionView</class>
   <extends>QWidget</extends>
   <header>IndirectSpectrumSelectionView.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
